app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/PortfolioCard.kt

```diff
@@ -66,7 +72,8 @@ fun PortfolioCard(
             modifier = Modifier
                 .constrainAs(todayProfitLabel) {
                     top.linkTo(totalCoins.bottom, margin = Dp40)
-                },
+                }
+                .testTag(tag = stringResource(R.string.portfolio_card_today_profit_label)),
```

Comment:
same, I think we should not use string resource for tag, let's say we have 2 different strings with the same value 🤓